api/location/schema.py

```diff
@@ -28,6 +29,10 @@ def mutate(self, info, **kwargs):
         # Validate if the country given is a valid country
         validate_country_field(**kwargs)
         validate_timezone_field(**kwargs)
+        query = Location.get_query(info)
+        result = query.filter(LocationModel.name == kwargs.get('name'))
+        if result.count() > 0:
```

Comment:
That was my initial approach, but checking if a result is available will always return the conflict message, because no result is still true. Thats why I checked for the count.  It was the best alternative.